lib/stripe.js

```diff
@@ -152,6 +153,7 @@ Stripe.errors = require('./Error');
 Stripe.webhooks = require('./Webhooks');
 
 Stripe.prototype = {
+  clientId: '',
```

Comment:
This could potentially be moved into an OAuth helper similar to `this._setApiField`, but that may be overkill for a single field.

Reply:
My preference would go towards exposing the client ID similarly to the API key, i.e. via a `setClientId` method.

Reply:
Although OAuth exists on a separate host from the API proper, would it make more sense to have the `clientId` stored in `_api` object or put it as a property on its own `_oauth` object?